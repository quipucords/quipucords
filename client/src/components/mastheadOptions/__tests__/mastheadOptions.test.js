import React from 'react';
import { mount } from 'enzyme';
import MastheadOptions from '../mastheadOptions';

describe('MastheadOptions Component', () => {
  it('should render', () => {
    const props = {
      user: { currentUser: { username: 'Admin' } }
    };

    const component = mount(<MastheadOptions {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
