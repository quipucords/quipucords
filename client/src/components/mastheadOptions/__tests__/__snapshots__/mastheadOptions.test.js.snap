// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MastheadOptions Component should render 1`] = `
<nav
  class="collapse navbar-collapse"
>
  <ul
    class="navbar-iconic nav navbar-nav navbar-right"
  >
    <li
      class="dropdown"
    >
      <a
        aria-expanded="false"
        aria-haspopup="true"
        class="nav-item-iconic dropdown-toggle"
        href="#"
        id="help"
        role="button"
      >
        <span
          aria-hidden="true"
          class="pficon pficon-help"
        />
         
        <span
          class="caret"
        />
      </a>
      <ul
        aria-labelledby="help"
        class="dropdown-menu"
        role="menu"
      >
        <li
          class=""
          role="presentation"
        >
          <a
            href="#"
            role="menuitem"
            tabindex="-1"
          >
            About
          </a>
        </li>
      </ul>
    </li>
    <li
      class="dropdown"
    >
      <a
        aria-expanded="false"
        aria-haspopup="true"
        class="nav-item-iconic dropdown-toggle"
        href="#"
        id="user"
        role="button"
      >
        <span
          aria-hidden="true"
          class="pficon pficon-user"
        />
         Admin 
        <span
          class="caret"
        />
      </a>
      <ul
        aria-labelledby="user"
        class="dropdown-menu"
        role="menu"
      >
        <li
          class=""
          role="presentation"
        >
          <a
            href="#"
            role="menuitem"
            tabindex="-1"
          >
            Log out
          </a>
        </li>
      </ul>
    </li>
  </ul>
</nav>
`;
